PyPDF2/generic.py

```diff
@@ -471,7 +471,7 @@ class NameObject(str, PdfObject):
     surfix = b_("/")
 
     def writeToStream(self, stream, encryption_key):
-        stream.write(b_(self))
+        stream.write(self.encode('utf8'))
```

Comment:
```suggestion
        stream.write(self.encode("utf8"))
```

Reply:
Could you please merge that tiny change? It's less about that change and more about making sure CI runs.

Reply:
I pulled in master and accidentally ran `black` across the file... I saw a commit doing that before, not sure why there are so many changes. 

For running the CI, seems like I need an approval from you 
![image](https://user-images.githubusercontent.com/1408472/162660273-22201ca7-43cb-49af-bb72-9d82836877b7.png)

Reply:
Thank you! I've approved and it seems that the 3.x versions are fine, but for the moment we also want to support 2.x